styles/variables-gallium.less

```diff
@@ -14,6 +14,8 @@
 
 // Checkbox
 // ---------------------------------------
+@agate-checkbox-size: 46px;
```

Comment:
Do we need to define it here because it is already defined with the same value in styles/variables-base.less (@agate-checkbox-size: 46px;)?

Reply:
No, I forgot to delete this. Thank you.